build.gradle

```diff
@@ -1,5 +1,6 @@
 plugins {
   id "com.gradle.plugin-publish" version "0.9.9"
+  id "java-gradle-plugin"
```

Comment:
@sic-bre can you please tell what this plugin `java-gradle-plugin` does?

Reply:
This plugin adds the necessary tasks to publish a plugin to a private Maven repository (which is what we have done for our "customized" version of the plugin behind the firewall). 
 For more details see https://docs.gradle.org/4.4.1/userguide/javaGradle_plugin.html

Reply:
Aha, ok, thanks for the explanation.

Reply:
Note: the user manual in the latest Gradle 4.5 was restructured, here is the plugin documentation from the latest release https://docs.gradle.org/4.5/userguide/java_gradle_plugin.html